OrbitGl/TracepointTrack.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_TRACEPOINT_TRACK_H_
+#define ORBIT_GL_TRACEPOINT_TRACK_H_
+
+#include "Track.h"
+
```

Comment:
Add includes for the obvious dependencies. As discussed before we are probably not great at this in the code base but let's try to keep this somewhat in shape.

Reply:
Thank you!